node/api/logs.go

```diff
@@ -39,6 +44,10 @@ type ContainerLogOpts struct {
 	Timestamps bool
 }
 
+func init() {
+	legacyscheme.Scheme.AddKnownTypes(v1.SchemeGroupVersion, &v1.PodLogOptions{})
```

Comment:
Let's not use `func init()`.

Also is there a way to not use a singleton like this?